time/timer.go

```diff
@@ -200,46 +202,42 @@ func NewTimerWheel() *TimerWheel {
 			if !w.enable.Load() {
```

Comment:
It seems that the new code 'enable' must be true, so should we consider removing this code?

Reply:
@pantianying no. pls check the enable in timer.Stop() func.